_job-scheduler-plugin/build-plugin.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+title: Building a plugin for Job Scheduler
+nav_order: 10
+---
+
+# Building a plugin for Job Scheduler
+
+OpenSearch plugin developers can extend the Job Scheduler plugin to schedule jobs like running aggregation query against raw data and saving the aggregated data into a new index every hour, or continue monitoring the shard allocation by calling the OpenSearch API and then post the output to a Webhook.
```

Comment:
Change verb tense at the end - "[...] and then posting the output to a webhook."

Also lowercase webhook